.envrc

```diff
@@ -0,0 +1 @@
+eval "$(lorri direnv)"
```

Comment:
Also, can you live these extra files out? you can put them in `.git/info/exclude`